saleor/order/actions.py

```diff
@@ -195,6 +196,26 @@ def cancel_order(
         call_event(send_order_canceled_confirmation, order, user, app, manager)
 
 
+def expire_order(
```

Comment:
In PR description we have `Saleor will use to change the unconfirmed and not paid orders into expired orders` maybe we should add validation for this function?

Reply:
I was also wondering about that. The idea of not having validation here is just for future development, maybe we will want to expire with different conditions. But I guess it will not harm to add that validation now.